company_website/static/main_page/main_screen.sass

```diff
@@ -0,0 +1,57 @@
+@import "partials/constants"
+.jumbo
```

Comment:
Please leave one space between import and class name.

Reply:
Please wrap all styles from this file in one class to avoid overwriting styles on classes from other files. Also please add this class to haml file to which this sass file is attached.